cmd/rosetta/cli.go

```diff
@@ -74,10 +74,16 @@ VERSION:
 		Value: strings.Repeat("0", 64),
 	}
 
-	cliFlagChainID = cli.StringFlag{
-		Name:  "chain-id",
-		Usage: "Specifies the Chain ID.",
-		Value: "1",
+	cliFlagNetworkID = cli.StringFlag{
+		Name:     "network-id",
+		Usage:    "Specifies the network ID (e.g. 1, D, T)",
+		Required: true,
+	}
+
+	cliFlagNetworkName = cli.StringFlag{
+		Name:     "network-name",
```

Comment:
We use rosetta's terminology here: blockchain name = elrond, network name = mainnet / devnet / testnet.